### ISSUE TITLE
IDs in .nwk and itol csv don't match

### ISSUE THREAD

**Issue by thackl** [BenMinch/NuPhylo#1]
Hi,

I'm trying to visualize the tree I got from runnning `NuPhylo -m PolB` on itol. But the IDs in the tree file (`((((GVMAG-M-3300009151-82.fltr_GVOGm0054.copy1:0.210693026,((GVMAG-M-3300027720-8.fltr_GVOGm0054, ...`) are different from those in the provided `NuPhylo_itol_colors.csv` (`Asfarviridae_AY261360_African_swine_fever_virus_isolate_Kenya_1950, ...`). How can I solve this?

Best
Thomas

**Comment by BenMinch** [comment]
Hey Thomas,

Thanks for reaching out! The itol color file is only meant to color the
reference sequences in the tree. By the names you provided it seems like
these are sequences from your dataset recovered by NCLDV marker search. I
typically leave these uncolored as they aren't "confirmed" from a certain
order before looking at the tree.

If you want to color them based on nearest neighbor, I would suggest
building a script to extract this information from the nwk file. I'm on
vacation this week but I can definitely try to make one when I get back.

Hope this helps,
Benjamin

**Comment by thackl** [comment]
Hi Benjamin,

thanks for the quick reply. I got the problem wrong. See the command I used below and the error in the 3rd line. It seems that NuPhylo can only find its own markers and add them to the tree when one runs NuPhylo from within the NuPhylo folder. Because I did not do that at first, there were no NuPhylo markers in my tree.

But I also used the wrong input file myself which included sequences from GVDB, which had GVDB IDs and I mistook for NuPhylo proteins...

Bottom line, the problem is fixed for me. But it might be useful to fix how NuPhylo ascertains the path to its marker file  to avoid the `cat` error below.

```
./NuPhylo/NuPhylo.py -i ggv0-polB.faa -m PolB -o gvv0-polB 
Aligning sequences
cat: Markers/PolB.final.faa: No such file or directory
outputhat23=16
```
